pgjdbc/src/main/java/org/postgresql/PGConnection.java

```diff
@@ -204,12 +204,14 @@
   String escapeLiteral(String literal) throws SQLException;
 
   /**
-   * Returns true if the connection is configured to use "simple 'Q' execute" commands only
-   * When running in simple protocol only, certain features are not available: callable statements,
+   * Returns the query mode for this connection.
```

Comment:
I'm not sure this is going in the right direction. We are providing less information in this version of the documentation than what was there. If anything providing a link or more description of the modes would be preferable to me.